internal/host/host.go

```diff
@@ -586,8 +588,13 @@ func (m *Manager) UpdateInstallProgress(ctx context.Context, h *models.Host, pro
 			swag.StringValue(h.Status), models.HostStatusError, statusInfo)
 	case models.HostStageRebooting:
 		if swag.StringValue(h.Kind) == models.HostKindAddToExistingClusterHost {
+			statusInfoMessage := statusInfoRebootingDay2
+			if m.kubeApiEnabled {
+				statusInfoMessage = statusInfoRebootingDay2ManagedInstallation
```

Comment:
We don't need any message here, the user doesn't need to do anything, just like they don't need to have a message / do anything for a regular rebooting stage during a normal day-1 installation